src/main/webapp/WEB-INF/view/activity-feed.jsp

```diff
@@ -29,6 +37,62 @@
 
         <h1>Welcome to the activity feed.</h1>
 
+        <div id = "container">
+        <ul>
+        <%
+        List<Conversation> conversations =
+          (List<Conversation>) request.getAttribute("conversations");
+
+        if(conversations != null && !conversations.isEmpty()) {
```

Comment:
You don't need to check if it's empty as if it is the for loop will simply never execute